cesium_app/tests/frontend/test_build_model.py

```diff
@@ -8,34 +8,54 @@
 from cesium_app.tests.fixtures import (create_test_project, create_test_dataset,
                                        create_test_featureset, create_test_model)
 
-test_model_name = str(uuid.uuid4())
 
+def _build_model(proj_id, model_type, driver):
+    driver.refresh()
+    proj_select = Select(driver.find_element_by_css_selector('[name=project]'))
+    proj_select.select_by_value(proj_id)
 
-def test_add_model(driver):
+    driver.find_element_by_id('react-tabs-6').click()
+    driver.find_element_by_partial_link_text('Create New Model').click()
+
+    model_select = Select(driver.find_element_by_css_selector('[name=modelType]'))
+    model_select.select_by_visible_text(model_type)
+
+    model_name = driver.find_element_by_css_selector('[name=modelName]')
+    test_model_name = str(uuid.uuid4())
+    model_name.send_keys(test_model_name)
+
+    driver.find_element_by_class_name('btn-primary').click()
+
+    try:
+        driver.implicitly_wait(0.5)
+        status_td = driver.find_element_by_xpath(
+            "//div[contains(text(),'Model training begun')]")
+
+        driver.implicitly_wait(2)
+        status_td = driver.find_element_by_xpath("//td[contains(text(),'Completed')]")
+    except:
+        driver.save_screenshot("/tmp/models_fail.png")
+        raise
+
+
+def test_build_model_rfc(driver):
     driver.get('/')
     with create_test_project() as p, create_test_featureset(p) as fs:
-        driver.refresh()
-        proj_select = Select(driver.find_element_by_css_selector('[name=project]'))
-        proj_select.select_by_value(str(p.id))
+        _build_model(str(p.id), 'RandomForestClassifier', driver)
```

Comment:
Perhaps call `str(id)` as the first line of `_build_model`, then these can disappear.